### ISSUE TITLE
python34 install it, still report 'Supervisor requires Python 2.4 or later but does not work!'

### ISSUE THREAD

**Issue by pamxy** [palmkevin/supervisor-py3k#2]
on debian
python3 setup.py install

and then show

`(myBlog) root@unicorn:/home/Workspace/Python/myBlogProject/install/supervisor-py3k-master# python3 setup.py install
Supervisor requires Python 2.4 or later but does not work on any version of Python 3.  You are using version 3.4.2 (default, Oct  8 2014, 10:45:20) 
[GCC 4.9.1].  Please install using a supported version.(myBlog) root@unicorn:/hosor-py3k-master# 
`

**Comment by palmkevin** [comment]
Unfortunately, this project is not really ready for use.
I ported the supervisor some years ago to python3 and I uploaded the code to this repo.
Notice that in my case the code is working (Py32 and Py35), but I didn't properly "install" the supervisor, but copied the files manually to a folder in the pythonpath.

I would recommend you to use the original [supervisor](https://github.com/Supervisor/supervisor) project.
They made a lot of progress in porting superisor to python 3. I evaluated in once and made [some issues.](https://github.com/Supervisor/supervisor/issues?q=is%3Aissue+is%3Aopen+label%3A%22python+3%22) .
Basically I think it is ready to be used in python3. I didn't have the. time yet to really switch in production back from my ported copy to the latest release of the official repo
